src/darwin/Framework/CHIP/include/CHIPProjectConfig.h

```diff
@@ -16,16 +17,9 @@
 
 /**
  *    @file
- *      Project configuration for Darwin Framework Tool.
+ *      CHIP project configuration for standalone builds.
```

Comment:
Is this really about "standalone" builds?

Reply:
What's going to be in this file that's not in `Darwin/SystemPlatformConfig.h` ?

Reply:
I’ve removed that file.